src/AutoCrudAdmin/Helpers/Implementations/FormControlsHelper.cs

```diff
@@ -78,12 +79,14 @@ public string GetComplexFormControlNameForEntityName(string entityName)
                         var value = ExpressionsBuilder.ForGetPropertyValue<TEntity>(
                             entityType.GetProperty(pair.Key))(entity);
 
+                        var isAutocompleteFormcontrol = property.PropertyType == typeof(UserProfile);
```

Comment:
This does not seem very extensible, we should have another way to determine if it's autocomplete property..

Reply:
Reworked.